be/src/vec/columns/column_string.h

```diff
@@ -62,13 +62,13 @@ class ColumnStr final : public COWHelper<IColumn, ColumnStr<T>> {
 
     static constexpr size_t MAX_STRINGS_OVERFLOW_SIZE = 128;
 
-    void static check_chars_length(size_t total_length, size_t element_number) {
+    void static check_chars_length(size_t total_length, size_t element_number, size_t rows = 0) {
         if (UNLIKELY(total_length > MAX_STRING_SIZE)) {
             throw Exception(
                     ErrorCode::STRING_OVERFLOW_IN_VEC_ENGINE,
                     "string column length is too large: total_length={}, element_number={}, "
-                    "you can set batch_size a number smaller than {} to avoid this error",
-                    total_length, element_number, element_number);
+                    "you can set batch_size a number smaller than {} to avoid this error. rows:{}",
+                    total_length, element_number, element_number, rows);
```

Comment:
It seems that ‘rows’ and ‘element_number’ have the same meaning

Reply:
at `insert_range_from` function
` check_chars_length(old_chars_size + nested_length, offsets.size() + length, size());
`
element_number is all size with need insert, size() is origin rows